tests/test-facebook-shortcode.php

```diff
@@ -48,6 +48,12 @@ public function test_pretty_permalink_video_display() {
 		$this->assertContains( '<div class="fb-post shortcake-bakery-responsive" data-href="https://www.facebook.com/coreycf/videos/953479961370562/"', apply_filters( 'the_content', $post->post_content ) );
 	}
 
+	public function test_photo_variant_display() {
+		$post_id = $this->factory->post->create( array( 'post_content' => '[facebook url="https://www.facebook.com/photo.php?fbid=2089627184639065/"]' ) );
```

Comment:
The trailing slash in the URL here seems to be extra, but it doesn't affect the functionality at all so no issue.